### PR TITLE
feature/homepage

### PR DESCRIPTION
**Pull request by wajeht**
First of all, thank you for your contribution! 😄

Please note that this repo is mostly for popular apps with thousands of stars and tens of thousands of downloads. If you'd like to add a less popular app, you can always [create your own 3rd party repo](https://github.com/caprover/one-click-apps#build-your-own-one-click-app-repository) and add your app there.

### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.
- [x] I've executed the checks if necessary by running `npm ci && npm run validate_apps && npm run formatter`
- [x] I will take responsibility addressing any issues that arises as a result of this PR (maintaining this app).

### CODE REVIEW

File: public/v4/apps/homepage.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          validRegex: /.{1,}/
+        - id: $$cap_homepage_version
+          label: Homepage Version
+          defaultValue: v0.7.4
+          description: Check out their GitHub packages page for the valid tags
+              https://github.com/gethomepage/homepage/releases
+          validRegex: /^([^\\s^\\/])+$/
+        - id: $$cap_config_path
+          label: Configuration Path
+          defaultValue: /path/to/config
```
Comment (githubsaturn) [inline public/v4/apps/homepage.yml:38]: Why don't use a volume name. It's more common and you don't need to make sure that the path exists

File: package-lock.json [hunk 2]
```diff
@@ -1,138 +1,182 @@
 {
   "name": "caprover-one-click-apps",
   "version": "1.0.0",
-  "lockfileVersion": 1,
```
Comment (githubsaturn) [inline package-lock.json:4]: Can you please revert the changes in package-lock?

**Review by githubsaturn — COMMENTED** [review]
Thanks!

File: public/v4/apps/homepage.yml (hunk 1 above)
Comment (wajeht) [reply public/v4/apps/homepage.yml]: done! still works. `https://test2.jaw.dev/`

**Comment by githubsaturn** [comment]
Amazing! Thanks!
